### ISSUE TITLE
about loglizer

### ISSUE THREAD

**Issue by F-SLoong** [shekharbiradar/CICIDS2017-GAN-BASED-ANOMALY-DETECTION#1]
Did you get results similar to those in the loglizer paper?

**Comment by shekharbiradar** [comment]
No, results that I got are very poor which makes loglizer code unusable. I think only unsupervised method makes sense in real world.
